tools/[rules].paths.ts

```diff
@@ -0,0 +1,73 @@
+// This script creates a Markdown document from the CDs Lint Rules Reference.
+// It uses VitePress' dynamic routing to create a single page.
+// See https://vitepress.dev/guide/routing#dynamic-routes for how this works
+
+export default {
+  async paths() {
+    const content = await getRules()
+    return [{ // we always just return one entry
+      params: { 'rules': 'rules' }, // must match to file name
+      content // raw MD
+    }]
+  }
+}
+
+
+async function getRules() {
+  try {
+    // @ts-ignore
+    await import('@sap/eslint-plugin-cds');
+
+    console.debug('Reading CDS Lint sources...')
+    let result = `Below you can find all rules in the \`@sap/eslint-plugin-cds\` ESLint plugin.
```

Comment:
```suggestion
    let result = `Below you can find all rules of the \`@sap/eslint-plugin-cds\` ESLint plugin.
```

Is it `in` or `of`? (I'm not sure here)

Reply:
Or `from`...? 🧐